roles/openshift_metrics/tasks/update_master_config.yaml

```diff
@@ -4,6 +4,9 @@
     dest: "{{ openshift.common.config_base }}/master/master-config.yaml"
     yaml_key: assetConfig.metricsPublicURL
     yaml_value: "https://{{ openshift_metrics_hawkular_hostname}}/hawkular/metrics"
-  notify: restart master
+  notify:
+  - restart master
+  - restart master api
```

Comment:
i was thinking since its currently copy/pasta, just to make it easier to rip it all out later, i'd keep the name... but i'm open to it